### ISSUE TITLE
Error in description of velocity

### ISSUE THREAD

**Issue by KatharinaDD** [cp2k/cp2k#3754]
Here https://github.com/cp2k/cp2k/blob/master/src/input_cp2k_motion_print.F#L110
and (vice versa) here: https://manual.cp2k.org/trunk/CP2K_INPUT/MOTION/PRINT/VELOCITIES.html,
it is stated that "Having an atom with a mass m in AMU the kinetic energy 1/2mv^2 will be obtained in Hartree (i.e. au) multiplying by 911.447". However, I believe this is incorrect. To me, it seems that C*m[amu]*v^2[atomic units] = E[Ha] for C=911.447, rather than  C*0.5*m[amu]*v^2[atomic units] = E[Ha] as stated in the text. 
My reasoning is as follows:
amu2kg * Bohr2^2 / atomic_units_of_time2s^2 / J2Ha =
= 1,660539040e-27*0,52917721067e-10^2/2.41888E-17^2/4.3597447222071e-18 = 
= 1,8228949765137519199e3
= 2C

**Comment by mkrack** [comment]
The description is not fully clear and can obviously be misunderstood. I think it is meant that the factor 911.447 includes already the pre-factor 1/2. Another deficiency is that the factor is only accurate to two decimal places. It should be 911.444 based on the conversion factors printed in each CP2K output run with `PRINT_LEVEL > low` which are the factors actually used in the calculation:
```
 *** Conversion factors ***

 [u] -> [a.u.]                                              1.82288848426455E+03
```
This factor multiplied by 1/2, the atomic mass in u, and the square of the corresponding atomic velocity should give the kinetic energy in a.u. for that atom.
Could you propose a better., i.e. unambiguously clear, description text?

**Comment by KatharinaDD** [comment]
I think I disagree with your second statement. 
When K[internal units] = 1/2 m[amu] v^2 [bohr^2/au of time ^2]
Then K[Ha] = K[internal units] * 1822.89 or
K[Ry] = K[internal units] * 911.447 but not
K[Ha] = K[internal units] * 911.447 (see calculation for the conversion factor above).
I may be mistaken, but my unit conversion looks correct to me. It may be a problem of Ry vs. Ha in the documentation.

**Comment by mkrack** [comment]
I do not see that we disagree. I stated that
K[Ha] = 1/2 * 1822.888 * m[amu] * v^2 [bohr^2/au of time ^2] = 911.444 * m[amu] * v^2 [bohr^2/au of time ^2]

**Comment by KatharinaDD** [comment]
Haha, Ok. I see. I just keep misunderstanding the written text! The following sentence is the one that puts me off the track: "This factor multiplied by 1/2, the atomic mass in u, and the square of the corresponding atomic velocity should give the kinetic energy in a.u. for that atom". I read this as C*0.5*m[amu]*v[bohr/atomic units of time]^2 = E, which is different from the equation you write above. I think the text should then read:
"This factor multiplied by the atomic mass in u, and the square of the corresponding atomic velocity should give the kinetic energy in a.u. for that atom" without the "multiplied by 1/2" part.

**Comment by mkrack** [comment]
Fixed by #3779.
